### ISSUE TITLE
[css-view-transitions-2] Feature detection for `@view-transition` rule

### ISSUE THREAD

**Issue by noamr** [w3c/csswg-drafts#9743]
Currently there is no (CSS or other) way to detect whether the `@view-transition` rule is supported by the browser.
Need to add something to the `@supports` and JS `CSS.supports` syntax.

**Comment by noamr** [comment]
Perhaps https://github.com/w3c/csswg-drafts/issues/2463 fixes this.

**Comment by SebastianZ** [comment]
Yes, according to the resolution in the other issue, support for `@view-transition` rules can be checked via `@supports at-rule(@view-transition)`.

(And my hope is that at-rule and descriptor feature detection will be specced and implemented _before_ we add more at-rules.)

Sebastian

**Comment by khushalsagar** [comment]
Sounds like this is fixed for all at-rules via https://github.com/w3c/csswg-drafts/issues/2463, so nothing specific needed here. Please reopen if I misunderstood.
